packages/common/src/types.ts

```diff
@@ -174,6 +175,15 @@ export type WalletMetadata = {
   universalProtocol?: {
     link: string;
   };
+  /**
+   * @desc 快捷扫码的参数是否支持
+   * @descEn Whether the parameters of fast scan code are supported
+   */
+  paramsFormat?: {
+    isSupport?: boolean;
+    tokenFormatStr?: string;
+    coinFormatStr?: string;
+  };
```

Comment:
感觉可以通过 `paramsFormat` 是否是 false 来判断支不支持快捷扫码
```suggestion
  paramsFormat?: false | {
    tokenFormatStr?: string;
    coinFormatStr?: string;
  };
```

Reply:
还有就是 `paramsFormat ` 这个名字好像跟注释中说的 “快捷扫码的参数是否支持” 并没有什么关联，容易误解